docs/templates/visualization.md

```diff
@@ -20,7 +20,7 @@ You can also directly obtain the `pydot.Graph` object and render it yourself,
 for example to show it in an ipython notebook :
 ```python
 from IPython.display import SVG
-from keras.utils import model_to_dot
+from keras.utils.vis_utils import model_to_dot
```

Comment:
No, it's meant to be imported from `utils.` vis_utils is not part of the public API